epoch-verifier/src/data/models.ts

```diff
@@ -180,3 +208,20 @@ export class EpochUqsImpl implements Deserializable<EpochUqs>{
   }
 }
 
+export class FinalizedSimulationImpl implements Deserializable<FinalizedSimulation>{
+  patronsRewards: bigint;
+  matchedRewards: bigint;
+  projectsRewards: SimulatedReward[];
+
+  from(input: any) {
```

Comment:
Whenever one writes `any` in TS, a tiny penguin dies somewhere.
Not blocking, just sad. ;).

Reply:
changed